client/README.md

```diff
@@ -0,0 +1,21 @@
+### Client
+
+Wrappers for the client used for making RPC calls with useful funcionalities like nonce tracking, multichain clients, ... 
+
+With it we can expose the contract call that we might use often using bindings from the bindings folder and objects from the crypto folder and do things like getting the hermes id from the registry smart contract, create transactions that execute smart contract calls, etc.
```

Comment:
We dont really expose as much as we wrap bindings together with a RPC client allowing to easily interact with them using the different clients: multchin, dryRun, regular etc.